src/custom/theme/cowSwapTheme.tsx

```diff
@@ -117,9 +117,9 @@ function themeVariables(darkMode: boolean, colorsTheme: Colors) {
       `,
     },
     appBody: {
-      boxShadow: `4px 4px 0px #000000;`,
+      boxShadow: `4px 4px 0px ${darkMode ? '#021e34' : '#000000'};`,
```

Comment:
boxShadow: `4px 4px 0px ${colorsTheme.black}`, as the black variable covers the darkMode color differences.